directanswercards/card_component.js

```diff
@@ -26,7 +26,6 @@ BaseDirectAnswerCard["{{componentName}}"] = class extends ANSWERS.Component {
 
     return super.setState({
       ...cardData,
-      feedbackEnabled: ANSWERS.getAnalyticsOptIn(),
```

Comment:
This line should be added back in. I think it got removed because the more recent fix was reverted after the original fix was re-reverted.